### DISCUSSION TITLE
Debian release request

### DISCUSSION THREAD

**Discussion by andrewbk** [himmelblau-idm/himmelblau#322] [General]
Hello,
Would it be possible to request a Debian package? Successful make on Debian 12.8 Bookworm but some dependencies come from unstable. Not sure if that is correct?
Thank you,
-Andrew

**Comment by dmulder** [comment]
I'll take a look at adding Debian.

**Comment by dmulder** [comment]
> some dependencies come from unstable. Not sure if that is correct?

That wouldn't surprise me. Debian uses really old package versions, and rarely updates them. There are some features in Himmelblau that require relatively new packages.

**Comment by dmulder** [comment]
See https://github.com/himmelblau-idm/himmelblau/pull/324

**Comment by dmulder** [comment]
https://github.com/himmelblau-idm/himmelblau/releases/tag/0.7.12

**Reply by andrewbk** [reply]
Thank you @dmulder !!

I will provide feedback shortly.
